examples/gameroom/database/src/helpers/gameroom.rs

```diff
@@ -116,6 +116,22 @@ pub fn gameroom_service_is_active(conn: &PgConnection, circuit_id: &str) -> Quer
         .or_else(|err| if err == NotFound { Ok(false) } else { Err(err) })
 }
 
+pub fn fetch_active_gamerooms(
```

Comment:
This should be renamed to make it clear it is fetching gameroom proposals, not gamerooms

Reply:
and services.

Reply:
This change could also be pulled into a separate commit, like the other query function in this PR